api/src/main/java/org/pmiops/workbench/cohortreview/ReviewQueryBuilder.java

```diff
@@ -107,6 +107,19 @@
       "from `${projectId}.${dataSetId}.%s`\n" +
       "where person_id = @" + NAMED_PARTICIPANTID_PARAM + "\n";
 
+  private static final String CHART_DATA_TEMPLATE =
+    "select distinct a.standard_name as standardName, a.standard_vocabulary as standardVocabulary, " +
+      "DATE(a.start_datetime) as startDate, a.age_at_event as ageAtEvent, rnk as rank\n" +
+      "from `${projectId}.${dataSetId}.%s` a\n" +
+      "left join (select standard_code, RANK() OVER(ORDER BY COUNT(*) DESC) as rnk\n" +
+      "from `${projectId}.${dataSetId}.%s`\n" +
+      "where person_id = @" + NAMED_PARTICIPANTID_PARAM + "\n" +
+      "group by standard_code\n" +
+      "LIMIT 5) b on a.standard_code = b.standard_code\n" +
```

Comment:
Should we move the 5 out as a variable since it shows up in 2 places here and it might change over time? Or even allow the UI to specify how many in a param?

Reply:
I can let the UI specify.. should we put a min/max on limit?

Reply:
Probably. how about Min = 1 and Max = 20

Reply:
done.